scripts/pylib/twister/twisterlib.py

```diff
@@ -2840,7 +2840,7 @@ def get_all_tests(self):
 
     @staticmethod
     def get_toolchain():
-        toolchain_script = ZEPHYR_BASE + "/cmake/verify-toolchain.cmake"
+        toolchain_script = Path(ZEPHYR_BASE) / Path('cmake/verify-toolchain.cmake')
         result = CMake.run_cmake_script([toolchain_script, "FORMAT=json"])
```

Comment:
Late to the party, but you are sure `run_cmake_script` can take a Path? You don't need to call `os.fspath(toolchain_script)` first?